pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -0,0 +1,62 @@
+{ buildNpmPackage, fetchFromGitHub, vencord, electron, pipewire, lib }:
+let
+  libPath = lib.makeLibraryPath [ pipewire ];
+in
+buildNpmPackage rec {
+  pname = "vesktop";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "Vencord";
+    repo = "Vesktop";
+    rev = "v${version}";
+    sha256 = "sha256-Njs3tACxUyRolYUtS/q2lITIQnUBFXVXWZEfQ66HpPM=";
+  };
+
+  prePatch = ''
+    cat ${./package-lock.json} > ./package-lock.json
+  '';
+
+  patches = [
+    ./use_system_vencord.patch
+  ];
+
+  postPatch = ''
+    sed -i "s!@VENCORD_STORE_DIR@!${vencord}!" src/main/constants.ts
+  '';
+
+  npmDepsHash = "sha256-X4/7N+l92CfTfD3XXifRd5G0oxGWvIUl8CG+foAyShc=";
+  npmFlags = [ "--legacy-peer-deps" ];
+  makeCacheWritable = true;
+  env.ELECTRON_SKIP_BINARY_DOWNLOAD = "1";
+
+  # this is consistent with other nixpkgs electron packages and upstream, as far as I am aware
+  # yes, upstream really packages it as "vesktop" but uses "vencorddesktop" file names
```

Comment:
I went to the releases tab and it looks like the linux files are packaged under `vencorddesktop`. I personally think we should follow [AUR](https://aur.archlinux.org/packages/vencord-desktop-git) and call it `vencord-desktop`.

Reply:
The project used to be called Vencord Desktop before it got renamed to Vesktop. I am sure sooner or later, the package on the AUR will be renamed/merged into `vesktop`

Reply:
Ah, okay.